modular_skyrat/modules/colony_fabricator/code/tools/tools.dm

```diff
@@ -98,7 +98,7 @@
 /obj/item/crowbar/large/doorforcer
 	name = "prybar"
 	desc = "A large, sturdy crowbar, painted orange. This one just happens to be tough enough to \
-		survive <b>forcing doors open</b>."
+		<b>force a door open</b>." //Bubber EDIT
```

Comment:
This is also not clear enough that the item breaks after one use, it can _very_ easily be interpreted as the exact same function as before (if they even notice the slight wording difference)

Reply:
I haven't had a chance to change it yet. I am currently away from home for work but I am going to make it resoundingly clear it breaks after one use.